src/data/phase/samhain.tsx

```diff
@@ -203,7 +203,11 @@ export const samhain = ({ samhain: { startDate, endDate } }: PhasesConfig): Phas
           contentDescription: (
             <p>
               Have a look on the{' '}
-              <a href="/builders" rel="noreferrer" target="_blank">
+              <a
+                href="https://nemeton.staging.okp4.network/builders/challenges#challenges"
```

Comment:
Please do not use absolute URL that refers to staging... 
What will happen in production environment?
You can use relative path because you are under routing here.